src/components/logged_out/LoggedOutWrapper.js

```diff
@@ -0,0 +1,114 @@
+import { createContext, useContext, useEffect, useState } from 'react';
+
+import WelcomeScreen from './WelcomeScreen';
+import LoginScreen from './LoginScreen';
+import AccountCreationScreen from './AccountCreationScreen';
+
+import { UserContext } from '../../App';
+
+/** 
+ * The LoginContext only exists when UserContext === null.
+ * Holds the user input, whether it's a match or not, as well as the action that the
+ * user is doing (to determine what screen to show).
+ */
+const LoginContext = createContext(null);
+
+const LoggedOutWrapper = () => {
+// Set the initial state of the loginContext.
+  const [loginState, setLoginState] = useState({
+    input: {
+      un: "",
+      pw: ""
+    },
+    match: {
```

Comment:
Do we need this `match` object? Unless I'm missing something it seems to be unused.

Reply:
You're right: i thought I would, but it turns out I was just able to use the stuff that I'd written in the APIs. I'll remove it.